src/components/LabelBase/styles.module.css

```diff
@@ -1,11 +1,19 @@
+:root {
+  font-size: 16px;
+}
```

Comment:
I think we can take this out and I'll include it in my PR that migrates the header